examples/multicluster/usecases/floating_nse_composition/cluster1/nse-passthrough-1/passthrough.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: nse-firewall-vpp
+  name: nse-passthrough-1
+spec:
+  selector:
+    matchLabels:
+      app: nse-firewall-vpp
+  template:
+    metadata:
+      labels:
+        app: nse-firewall-vpp
+        "spiffe.io/spiffe-id": "true"
+    spec:
+      containers:
+        - name: nse
+          image: ghcr.io/networkservicemesh/ci/cmd-nse-firewall-vpp:5d66abd
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: NSM_SERVICE_NAME
+              value: "nse-composition@my.cluster3"
```

Comment:
I think it's better to use `interdomain-nse-composition@my.cluster3` to be consistent with other examples.